src/sql/src/pure/mysql.rs

```diff
@@ -342,19 +342,17 @@ pub(super) async fn purify_source_exports(
     ignore_columns: Vec<UnresolvedItemName>,
     unresolved_source_name: &UnresolvedItemName,
     initial_gtid_set: String,
+    allow_empty_references: bool,
```

Comment:
Can we have a proper enum for this parameter? Bare `bool`s are easy to mishandle